in_progress/roles.md

```diff
@@ -0,0 +1,109 @@
+# Abstract
+
+The purpose of this document is to formalize the governance process used by the `scverse` project, to clarify how decisions are made and how the various elements of our community interact.
+
+As a whole, we are a consensus-based community project.
+However, any subproject within the scverse can have their own governance system as long as they do not conflict with the rules set out here.
+Anyone with an interest in the project can join the community, contribute to the project design, and participate in the decision making process.
+This document describes how that participation takes place, how to find consensus, and how deadlocks are resolved.
+
+# Roles And Responsibilities
+
+## The Community
+
+The scverse community consists of anyone using or working with the project in any way.
+
+## Contributors/ Project developers
+
+A community member can become a contributor by interacting directly with the project in concrete ways, such as:
+
+- proposing, discussing, or reviewing a change to the code, documentation, or specification via a GitHub pull request to any https://github.com/scverse repository;
+- reporting a GitHub issue to any https://github.com/scverse repository;
+- discussing examples or usage issues on any of our discussion platforms
+
+among other possibilities.
+Any community member can become a contributor, and all are encouraged to do so.
+By contributing to the project, community members can directly help to shape its future.
+
+All community members are required to adhere by our code of conduct (**TODO: add link**).
+Potential contributors are encouraged to read the Contributing Guide (**TODO: add link**).
+
+A community member becomes a contributor when the following criteria are met:
+
+- At least two core developers from separate employers agree membership is a good idea.
```

Comment:
Should we state here how to apply for a developer position? Ideally I would almost hope that we could automate this in some way in the future...

Reply:
"core developer" or "contributor"?

Contributor, probably private chat channel, voting w/  :+1: :-1:? Bots seem easy enough, but it would have to know about employers for this.

Reply:
Think that the "separate employers" is over complicating things. It's not like this is a super serious role. I'd suggest not to overengineer this.

Reply:
I think this could take a little more thought, but not super important to happen right now. Some thoughts:

* I think a little friction is good for allowing anyone to raise an objection (e.g. history the rest of the team may not know about – probably also the only case where someone would get rejected?)
* I like the multiple employers thing is good for preventing a whole lab being added in a day of work that "is going to happen"
* I would also like to consider a simple onboarding for contributors, basically welcomes and introductions. Make new people feel welcome.

Reply:
> I think a little friction is good for allowing anyone to raise an objection (e.g. history the rest of the team may not know about – probably also the only case where someone would get rejected?)

Sure, agreed. I think that the likelihood of this happening is incredibly small based on my experience with the other communities.

> I like the multiple employers thing is good for preventing a whole lab being added in a day of work that "is going to happen"

Sure, I got that. But why not? If we want to bring in and encourage as many people as possible this process should be as easy and fast as possible. In nf-core you get commit rights by writing your Github username into a Slack channel. Everyone gets invited. We don't have to follow this model, but I would suggest to make things as easy and fast as possible. Everything else is overhead that discourages people to contribute and results in more work for us.

> I would also like to consider a simple onboarding for contributors, basically welcomes and introductions. Make new people feel welcome.

Yeah. Think that this depends on our chat solution, but a section on the website would also be nice. :+1: